samples/nextjs/.env

```diff
@@ -24,6 +24,12 @@ JSS_EDITING_SECRET=
 # In this case, use this environment variable to provide the value at build time.
 SITECORE_API_KEY=
 
+# An SSC API Key which has been configured for use with
+# the Tracking Service (Sitecore.Analytics.Controllers.TrackController, Sitecore.Analytics)
+# and Personalization Service controllers (Sitecore.LayoutService.Mvc.Controllers.LayoutServiceController, Sitecore.LayoutService.Mvc).
+# This key is required when using Sitecore Experience Edge and personalization/tracking powered by Sitecore Content Delivery instance.
+SITECORE_SERVICES_API_KEY=
```

Comment:
It is not. :-)
In case the Experience Edge is used for content delivery and Sicore CD for personalization decision making two keys are required. This is approved by Product Management.